rocky/rocky/views/organization_member_add.py

```diff
@@ -54,3 +69,89 @@ def form_valid(self, form):
     def add_success_notification(self):
         success_message = _("Member added successfully.")
         messages.add_message(self.request, messages.SUCCESS, success_message)
+
+
+class MembersUploadView(OrganizationPermissionRequiredMixin, OrganizationView, FormView):
+    template_name = "organizations/organization_member_upload.html"
+    form_class = UploadCSVForm
+    permission_required = "tools.add_organizationmember"
+
+    def get_success_url(self):
+        return reverse_lazy("organization_member_list", kwargs={"organization_code": self.organization.code})
+
+    def form_valid(self, form):
+        self.process_csv(form)
+        return super().form_valid(form)
+
+    def process_csv(self, form) -> None:
```

Comment:
To me most of these log statements here are more error logs than exception logs

Reply:
I would always want the exception info to be printed in these cases as they would be probably be unexpected database exceptions?

Reply:
> What do you mean when you say too broad? The exception clause or the message?

I'm referring to the exception clause indeed.

> I have looked into catching just Django related database errors here, but there is no real base exception for that to catch, so that would mean having a list of exceptions here. But then again the argument could be made that we do not want to handle unexpected database exceptions here and just crash, since such information would not be propagated to the user uploading the file and does not give information about existence of the users in the file. I think I actually prefer that and we can actually not continue on such errors.
> 
> (Still need to "roll back" the creation of the user though.)

Maybe I wasn't clear, but I meant to use `logger.error` more instead of the `logger.exception` (in this case and context). Catching the exceptions (and e.g. perform roll back actions) is obviously properly implemented.

Reply:
@ammar92 Can you have another look? I cleaned up the error handling